core/store/src/genesis/initialization.rs

```diff
@@ -42,6 +43,17 @@ pub fn initialize_sharded_genesis_state(
         get_genesis_state_roots(&store)
             .expect("Store failed on genesis initialization")
             .expect("Genesis state roots not found in storage");
+        // TODO: with 2.6 release, remove storing genesis height
```

Comment:
This is needed for initializing the genesis_height. I tried going the DB migration approach, but that's not possible as we unfortunately don't save the genesis anywhere and this change was simple enough to implement.

Reply:
Where is this function called? Is it part of node startup?

Reply:
Yes, this is called every time the node starts